### PR TITLE
Scheduling session start

### PR DESCRIPTION
**Pull request by amoscalamida**

### CODE REVIEW

File: src/components/overview/Overview.js [hunk 1]
```diff
@@ -1,16 +1,120 @@
...
+            yourSelection: yourSelection
+        })
+    }
+
+    handleYourCreation(yourCreation) {
+        //called in "receive()"
+        this.setState({
+            yourCreation: yourCreation
+        })
+    }
```
Comment (MESC2020) [inline src/components/overview/Overview.js:88]:
```suggestion
 
```
We probably should delete this if we push it on Master

File: src/components/overview/Overview.js [hunk 2]
```diff
@@ -1,16 +1,120 @@
...
+                    placeholder={"time"}
+                    fullWidth={true}
+                    onChange={e => this.handleInputChange('time', e.target.value)}
+                />
+                <Button
+                    disabled={!this.state.date || !this.state.time}
+                    onClick={() => {
+                        this.handleMyCreation()
+                    }}> propose
+                </Button>}
```
Comment (MESC2020) [inline src/components/overview/Overview.js:117]:
```suggestion

```

File: src/components/overview/Overview.js [hunk 3]
```diff
@@ -1,16 +1,120 @@
...
+        this.setState({
+            yourCreation: yourCreation
+        })
+    }
+
 
     render() {
         return (
             <div>
+                <NewWebRTC/>
```
Comment (MESC2020) [inline src/components/overview/Overview.js:94]: Would you like to keep this in **Overview**?

File: src/helpers/getDomain.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ import {isProduction} from './isProduction';
  */
 export const getDomain = () => {
   const prodUrl = 'https://sopra-fs21-group-16-server.herokuapp.com';
-  const devUrl = 'http://localhost:8080';
+  const devUrl = 'http://192.168.178.172:8080';
```
Comment (MESC2020) [inline src/helpers/getDomain.js:11]:
```suggestion
  const devUrl = 'http://localhost:8080';
```

**Review by MESC2020 — CHANGES_REQUESTED** [review]
- Should clean up **Overview**, what do you think?
- changed IP back to localhost

File: src/components/overview/Overview.js (hunk 3 above)
Comment (amoscalamida) [reply src/components/overview/Overview.js:22]: Yes just to test.

**Review by MESC2020 — APPROVED** [review]

**Pull request merged**